x-pack/plugins/uptime/server/lib/saved_objects/synthetics_monitor.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsType } from 'kibana/server';
+import { i18n } from '@kbn/i18n';
+
+export const syntheticsMonitorType = 'synthetics-monitor';
+
+export const syntheticsMonitor: SavedObjectsType = {
+  name: syntheticsMonitorType,
+  hidden: false,
+  namespaceType: 'single',
+  mappings: {
+    dynamic: false,
+    properties: {
```

Comment:
Some properties are missing for things like `tags` and `schedule`. At least tags should be indexed, as I assume we'll want to search by tags.

Reply:
i have updated to add tags